code/game/machinery/computer/arcade.dm

```diff
@@ -543,7 +543,7 @@ GLOBAL_LIST_INIT(arcade_prize_pool, list(
 			playsound(loc, 'sound/arcade/win.ogg', 50, TRUE)
 
 			if(obj_flags & EMAGGED)
-				new /obj/effect/spawner/newbomb/timer/syndicate(loc)
```

Comment:
Is this meant to be weaker or something? I'm kinda confused about this typepath to be honest

Reply:
The syndicate subtype was basically just a maxcap subtype for bomb spawners. (eg: It produced a maxcap rather than the weaker bombs the others did.) Since it was the only subtype that was actually used I nixed it and just made bomb spawners default to making maxcaps.

Reply:
👍